internal/lxdutils/lxd.go

```diff
@@ -203,7 +203,8 @@ func containerName(username string) string {
 	if len(name) > 60 {
 		name = name[:60]
 	}
-	return name
+	// Container names must end with letters or digits.
+	return strings.TrimRight(name, "-")
```

Comment:
should we leave anything that's not a normal character perhaps? I'm wondering if the best thing is to find the last alphanum and then use that as the break point?

Reply:
I think that we already replace all the invalid characters before in that function. But trimming all non-alpha could be a good idea so that we are sure-sure